ShovIP/app/src/main/java/com/example/shovip/DialPadFragment.kt

```diff
@@ -2,19 +2,21 @@ package com.example.shovip
 
 import android.annotation.SuppressLint
 import android.content.Context
+import android.graphics.Color
 import android.os.Bundle
 import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.core.content.ContextCompat
 import androidx.navigation.fragment.findNavController
 import com.example.shovip.databinding.FragmentDialPadBinding
 
 /**
  * A simple [Fragment] subclass as the default destination in the navigation.
  */
 class DialPadFragment : Fragment() {
-    private var binding by autoCleared<FragmentDialPadBinding>()
+    private lateinit var binding : FragmentDialPadBinding
```

Comment:
@shovian 

What was the motivation for changing this? AFAIR, the comment had already been resolved.

As Yamauchi-san mentioned, the binding in the fragment should *not* be `lateinit` after all/

@atsushi-ageet Please double check.